src/Crawler.php

```diff
@@ -59,7 +64,8 @@ public function __construct(
         int $limit,
         string $userAgent,
         int $cacheMethod = Recorder::CACHE_ID,
-        int $waitInMicroSeconds = 100000
+        int $waitInMicroSeconds = 100000,
+        string $dataDirectoryBasePath = null
```

Comment:
Add same attribute for CrawlerContinue::_constructor to give full compatibility for this new feature.

Reply:
and for CrawlerRestart::__construct too.